packages/third-parties/README.md

```diff
@@ -0,0 +1,44 @@
+# `@next/third-parties`
+
+`@next/third-parties` is collection of components that can be used to efficiently load third party libraries into your Next.js application.
+This package uses [Third Party Capital](https://github.com/GoogleChromeLabs/third-party-capital/) under the hood to fetch the best loading recommendations.
+
+> Note: `@next/third-parties` is still experimental and under active development
+
+# Usage
+
+## Google third parties
+
+### Youtube Embed
+
+The `YoutubeEmbed` component is used to load and display a YouTube embed. This component loads faster by using [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed) under the hood.
+
+```js
+import { YoutubeEmbed } from '@next/third-parties/google'
+
+export default function Page() {
+  return <YoutubeEmbed videoid="ogfYd705cRs" height={400} />
```

Comment:
Sure

Reply:
Since you're using version 1.0.18 now, I think it's easier if we just update the export naming to `YouTubeEmbed` now to match Third Party Capital instead of pushing out a new PR

Reply:
It looks like we downgraded back to `1.0.17` in this PR. I think the plan is to do this in a follow-up PR once upgrading to the latest version.